apps/hash-ai-worker-ts/src/activities/flow-activities/research-entities-action/deduplicate-entities.ai.test.ts

```diff
@@ -114,7 +114,7 @@ const nonDuplicateEntitySummaries: LocalEntitySummary[] = [
   },
 ];
 
-test(
+test.skip(
```

Comment:
Is this intentionally skipped?

Reply:
I'm using these test files to test out the methods, and am skipping/not skipping tests based on which ones I want to run. I appreciate this adds things to the diff unnecessarily depending on the last test I ran whilst developing, if you'd like maybe I can mark all tests as skipped and manually unskip them without committing in development. Might be an extra layer of safety to prevent someone unintentionally running AI tests.

Reply:
Maybe yes, and some comment explaining what these are for. Can be in follow up. A lot of them just expect the response to be defined which doesn't seem super useful but I may be misunderstanding still

Reply:
I've skipped the tests in https://github.com/hashintel/hash/pull/4505/commits/513f37af6044f441c4d2210769bb96c3305f4dc6.

Sorry if this hasn't been clear, but these "tests" aren't defined to be tests that can pass or fail, they should probably be thought of as scripts which I use to easily run methods with different inputs, observe their logs, and read they're outputs (manually in the console). The reason I used a testing framework for this is to make the process of mocking methods such as `getFlowContext` more easily, so that running these doesn't require creating a flow entity, temporal workflow, etc. I've been committing them because they are useful to be able to return back to, and not have to define from scratch when developing a method.